core/scripts/fetch-test-data.sh

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env bash
-cd ..
-git submodule update --init --recursive
\ No newline at end of file
+
+CURRENT_DIR=`pwd`
+TISPARK_HOME="$(cd "`dirname "$0"`"/../..; pwd)"
```

Comment:
consider move this `TISPARK_HOME` to an `_env.sh` so that all bash scripts can source the environment settings